web/client/codechecker_client/cmd_line_client.py

```diff
@@ -481,14 +481,15 @@ def skip_report_dir_result(report):
 
         return False
 
-    def get_report_dir_results(reportdir):
+    def get_report_dir_results(report_dirs):
```

Comment:
Earlier this function got an absolute path as parameter. At each invocation the original `reportdir` parameter was an absolute path. Now this is a list of (not necessarily absolute) paths. I think it can be correct, or at least I can't see if this would be wrong. I just noticed this change.

Reply:
The `preprocess_run_arg` will process the given arguments and convert report directories to absolute path.

Reply:
Please mention in the function documentation that absolute paths are expected in the `report_dirs`.